synapse/storage/stream.py

```diff
@@ -233,52 +233,49 @@ def get_rooms_that_changed(self, room_ids, from_key):
     @defer.inlineCallbacks
     def get_room_events_stream_for_room(self, room_id, from_key, to_key, limit=0,
                                         order='DESC'):
-        # Note: If from_key is None then we return in topological order. This
```

Comment:
I think it might be worth preserving this comment, but moving it to sync.py?

Reply:
Looking at it I don't really know why we don't always use the topological ordering, but I've added the comment.